prisma/schema.prisma

```diff
@@ -11,11 +11,19 @@ datasource db {
   relationMode = "prisma"
 }
 
+enum Role {
```

Comment:
An enum enforces the value to be one of the different options. Inserting any other value will result in an error.